src/app/modules/home/home.module.ts

```diff
@@ -2,11 +2,13 @@ import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
 import {HomeRoutingModule} from './home-routing.module';
 import {HomeComponent} from './home.component';
+import { ConsultationCardModule } from 'src/app/shared/components/consultation-card/consultation-card.module';
```

Comment:
@barath46 Consultation card should just be a component. Where as the consultations listing should be the module